docs/worker-node/install-singularity.md

```diff
@@ -40,10 +45,11 @@ As with all OSG software installations, there are some one-time (per host) steps
 - Obtain root access to the host
 - If you're installing singularity, prepare the [required Yum repositories](../common/yum)
 
-## Using Singularity via CVMFS (EL 7.4 only)
+## Using Singularity via CVMFS (EL 7 only)
 
-If you are using a RHEL 7.4 variant OS, you can skip installation
-altogether and use singularity through CVMFS:
+If the operating system is an EL 7 variant, and has been updated to EL
+7.4 or the 7.4 kernel (3.10.0-693 or greater), you can skip
+installation altogether and use singularity through CVMFS:
```

Comment:
How about if we get rid of the `(EL 7 only)` text in the header and add a note to the top of the section:

```
!!! note
    The instructions in this section require kernel version >= 3.10.0-693
```

Reply:
I think it's helpful to at least make it clear to people at a glance that any EL6 kernel won't do what they need. 

I don't fully envision the changes that you're asking for, so I suggest that you go ahead and update it to how you'd like it and push that to the pull request.

Reply:
Fair enough